src/Sylius/Bundle/PayumBundle/Resources/config/services/action.xml

```diff
@@ -27,7 +27,7 @@
         <service id="sylius.payum_action.offline.convert_payment" class="Sylius\Bundle\PayumBundle\Action\Offline\ConvertPaymentAction">
             <tag name="payum.action" factory="offline" alias="sylius.offline.convert_payment" />
         </service>
-        <service id="sylus.payum_action.offline.status" class="Sylius\Bundle\PayumBundle\Action\Offline\StatusAction">
+        <service id="sylius.payum_action.offline.status" class="Sylius\Bundle\PayumBundle\Action\Offline\StatusAction">
```

Comment:
While it's a typo for sure, it might be a BC break as well (people can already rely on this name). I'll look into ways of deprecating the current service name and introducing the right one.